examples/todolist/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  testcafe:
+    build:
+      context: .
+      dockerfile: testcafe.dockerfile
+    command:
+      - firefox
```

Comment:
Yes, where is chrome / chromium? Yes, it can't run with default options, but can be started with --no-sandbox option